src/util.ts

```diff
@@ -35,7 +35,7 @@ export interface ICommandAction {
     description?: string;
     help?: string;
     params: string[];
-    permission?: PERMISSIONTYPES;
+    permission?: PERMISSIONTYPES | PERMISSIONTYPES[];
```

Comment:
eeeer, wouldn't it be better to change `type PERMISSIONTYPES = any | any[]; // tslint:disable-line no-any` instead?

Reply:
oh yeah, maybe